client/src/amplify-ui/block-switcher/block-switcher.spec.ts

```diff
@@ -1,10 +1,26 @@
 import {AmplifyBlockSwitcher} from "./block-switcher";
+import {AmplifyCodeBlock} from "./block";
 import {newSpecPage} from "@stencil/core/testing";
 
 describe("amplify-block-switcher", () => {
   describe("Component logic", () => {
-    let blockSwitcher: AmplifyBlockSwitcher;
```

Comment:
Why are removing this?

Reply:
that's a good question...

Reply:
My logic at the time was that it's just unused boilerplate code (and that having it around creates a name conflict), but I could go either way on it.  What do you think, @sammartinez?

Reply:
Fair logic, We do use it as a new instance for each test is ran. Can we validate it doesn't break exisiting test?